cfg/defaultConstants.js

```diff
@@ -1,27 +1,41 @@
 module.exports = {
 
-    baseUrl: JSON.stringify("http://localhost:50000/"),
+  baseUrl: 'http://localhost:50000/',
 
-    authRedirectUri: JSON.stringify("http://localhost:3000/"),
+  authRedirectUri: 'http://localhost:3000/',
 
-    retries: 2,
+  retries: 2,
 
-    retryExponentialBackoffFactor: 2,
+  retryExponentialBackoffFactor: 2,
 
-    retryMinTimeoutInMiliseconds: 50,
+  retryMinTimeoutInMiliseconds: 50,
 
-    retryMaxTimeoutInMiliseconds: 1500,
+  retryMaxTimeoutInMiliseconds: 1500,
 
-    ticketRefreshIntervalInSeconds: 300,
+  ticketRefreshIntervalInSeconds: 300,
 
-    authVersion: "'ADAL'",
+  authVersion: 'ADAL',
 
-//   Leave this as is if your code is in a public repo (or delete it if you want).  You can override the defaults with the real information in your cfg/$env.const.js file.  
+  //   Leave this as is if your code is in a public repo (or delete it if you want).
+  // You can override the defaults with the real information in your cfg/$env.const.js file.
+  aadInstance: 'https://login.microsoftonline.com/', // Replace only if using AAD non-public instances.
 
-    aadInstance: "'https://login.microsoftonline.com/'", //Replace only if using AAD non-public instances. Yes, use both single and double quotes.
+  aadTenant: 'Your Azure Active Directory tenant here',
 
-    aadTenant: "'Your Azure Active Directory tenant here'", //Yes, use both single and double quotes.
-
-    clientId: "'Your EventUI Azure Active Directory app registration client ID here'" //Yes, use both single and double quotes.
+  clientId: 'Your EventUI Azure Active Directory app registration client ID here',
 
+  ticketSystems: {
```

Comment:
The PowerShell configuration builder script needs to be updated to be consistent with the double quotes changes and allow for generating a ticketing system from environment variables before we merge this.

Reply:
on it...

Reply:
Done!

Reply:
I updated the build with the new environment variables, so we're ready to merge.